internal/controllers/topology/cluster/patches/engine_test.go

```diff
@@ -361,7 +366,17 @@ func TestApply(t *testing.T) {
 			blueprint, desired := setupTestObjects()
 
 			// If there are patches, set up patch generators.
-			patchEngine := NewEngine()
+			// FIXME(sbueringer) implement test for external patches
```

Comment:
Current version is still missing these tests.

Reply:
Just to clarify: Plan is to add them in a follow-up PR which will be up today or tomorrow